README.md

```diff
@@ -166,108 +151,100 @@ report=true
-pluginsync = true
+1. Edit your `/etc/puppetlabs/puppet/puppet.conf` file to add the puppet Agent:
 
-[agent]
-report = true
-pluginsync = true
-```
+    ```ini
+    [master]
+    report = true
+    reports = datadog_reports
+    pluginsync = true
 
-_Note: This may be file `/etc/puppet/puppet/puppet.conf` on older puppets_
+    [agent]
+    report = true
+    pluginsync = true
+    ```
 
-/etc/puppetlabs/code/environments/production/manifests/10_nodes.pp
-------------------------------
+     _Note: This may be file `/etc/puppet/puppet/puppet.conf` on older puppets_
 
-```
-node "default" {
-    class { "datadog_agent":
-        api_key => "<YOUR_DD_API_KEY>",
+2. Edit your `/etc/puppetlabs/code/environments/production/manifests/10_nodes.pp` file to configure your Agent:
+
+    ```conf
+    node "default" {
+        class { "datadog_agent":
+            api_key => "<YOUR_DD_API_KEY>",
+        }
     }
-}
-node "puppetmaster" {
-    class { "datadog_agent":
-        api_key            => "<YOUR_DD_API_KEY>",
-        puppet_run_reports => true
+    node "puppetmaster" {
+        class { "datadog_agent":
+            api_key            => "<YOUR_DD_API_KEY>",
+            puppet_run_reports => true
+        }
     }
-}
-```
-
-_Note: This may be file `/etc/puppet/manifests/nodes.pp` on older puppets_
-
+    ```
 
-Run Puppet Agent
-----------------
+     _Note: This may be file `/etc/puppet/manifests/nodes.pp` on older puppets_
 
-```
-sudo systemctl restart puppetserver
-sudo puppet agent --onetime --no-daemonize --no-splay --verbose
-```
+3. Run Puppet Agent
 
-Example response:
-
-```
-info: Retrieving plugin
-info: Caching catalog for alq-linux.dev.datadoghq.com
-info: Applying configuration version '1333470114'
-notice: Finished catalog run in 0.81 seconds
-```
+    ```shell
+    sudo systemctl restart puppetserver
+    sudo puppet agent --onetime --no-daemonize --no-splay --verbose
+    ```
 
-Verify on Datadog
------------------
+     Example response:
 
-1. Search for `puppet` on [the Integrations page][5]. The Puppet integration tile displays the install status.
+    ```text
+    info: Retrieving plugin
+    info: Caching catalog for alq-linux.dev.datadoghq.com
+    info: Applying configuration version '1333470114'
+    notice: Finished catalog run in 0.81 seconds
+    ```
 
-2. Search for `sources:puppet` in the [Event Stream][6] to see your Puppet events.
+4. Verify on Datadog
 
-Masterless Puppet
-=================
+    1. Search for `puppet` on [the Integrations page][4]. The Puppet integration tile displays the install status.
+    2. Search for `sources:puppet` in the [Event Stream][5] to see your Puppet events.
 
-To use this specific setup, see https://gist.github.com/LeoCavaille/cd412c7a9ff5caec462f. This applies to older puppets and is untested on >=4.x Puppet versions.
+## Masterless Puppet
 
+To use this specific setup, see this [Github gist][6]. This applies to older puppets and is untested on >=4.x Puppet versions.
```

Comment:
````suggestion
1. The Datadog module and its dependencies have to be installed on all nodes running masterless.
2. Add this to each node's `site.pp` file:
    ```conf
    class { "datadog_agent":
        api_key            => "your_api_key_here",
        puppet_run_reports => true
    }
   ```

3. Configure reports in the `[main]` section of `puppet.conf`:
    ```conf
    [main]
    reports=datadog_reports
    ```
4. Run puppet in masterless configuration:
    ```shell
    puppet apply --modulepath <path_to_modules> <path_to_site.pp>
    ```
````
Up to date instructions.